frontend-react/src/pages/error/ErrorPage.tsx

```diff
@@ -1,106 +1,32 @@
-import { Alert } from "@trussworks/react-uswds";
 import React from "react";
-import Helmet from "react-helmet";
 
-import { NotFound } from "./NotFound";
-import { UnsupportedBrowser } from "./UnsupportedBrowser";
+import { GenericError } from "./legacy-content/Generic";
 
-/* INFO
-   For consistency, when passing the code prop, please use these values
-   e.g. <ErrorPage code={CODES.NOT_FOUND_404} />
- */
-export enum CODES {
-    UNSUPPORTED_BROWSER = "unsupported-browser",
-    NOT_FOUND_404 = "not-found",
-    UNKNOWN = "unknown-error",
+/** Config to suit page-style templates */
+export interface ErrorPageContentConfig {
+    header: string;
+    paragraph: string;
 }
+/** Union type for  */
+export type ErrorDisplayConfig = ErrorPageContentConfig | string;
 
-interface ErrorPageProps {
-    code?: CODES;
-    error?: string;
-    errorInfo?: React.ErrorInfo;
-    type?: "page" | "message";
-}
-
-const ErrorPageWrapper = (props: React.PropsWithChildren<ErrorPageProps>) => {
+/** Just  */
+export const ErrorDisplayWrapper = (props: React.PropsWithChildren<{}>) => {
     return (
-        <div className="usa-section padding-top-6">
-            <div className="grid-container">
-                <div className="grid-row grid-gap">{props.children}</div>
-            </div>
+        <div data-testid={"error-display-wrapper"} className="grid-container">
```

Comment:
Yep. By the end of this transition, we won't be using this. I'll move the `div` into RSErrorBoundary and it'll be deprecated. It's still out here because of the split use of ErrorPage. Moving forward, error ui's manage their own grid-container.

Reply:
Basically this will be inside of RSErrorBoundary:

```typescript jsx
<div className="grid-container">
      <GenericError displayConfig={config} displayAsPage={type === "page"} />
 </div>
```

We'll be able to drop the wrapper AND `ErrorPage` altogether since we don't have cases for displaying an error without throwing an error to be caught by the boundary.